feast-ui/rockcraft.yaml

```diff
@@ -53,30 +51,24 @@ parts:
       make build-ui
 
       # Replace the build artifacts inside the Python package
-      rm -rf /usr/local/lib/python3.10/dist-packages/feast/ui/build/
-      cp -r sdk/python/feast/ui/build /usr/local/lib/python3.10/dist-packages/feast/ui/build/
+      rm -rf /usr/local/lib/python3.12/dist-packages/feast/ui/build/
+      cp -r sdk/python/feast/ui/build /usr/local/lib/python3.12/dist-packages/feast/ui/build/
+
+      # Patch Feast CLI to explicitly use python3.12 interpreter
+      sed -i '1s|.*|#!/usr/bin/python3.12|' /usr/local/bin/feast
```

Comment:
suggestion: make the `sed` command more persistent by specifying the text to replace instead of the line number

Reply:
since this is the binary we will always need to change the first line where it specifies the python version to be used